### ISSUE TITLE
HEVC in Safari

### ISSUE THREAD

**Issue by vasimi** [Fyrd/caniuse#3521]
According to WWDC 2017, In macOS High Sierra and iOS 11, Safari supports HEVC natively.

**Comment by daytonlowell** [comment]
https://github.com/Fyrd/caniuse/pull/3561

**Comment by Schweinepriester** [comment]
I believe this issue can be closed due to #3561 => <https://caniuse.com/#feat=hevc>?

**Comment by Themanwithoutaplan** [comment]
HEVC is supported in Safari on High Sierra but doesn't seem to coexist with h264 which Safari will play in preference.
```
<video autobuffer="true" autoloop="" loop="" controls="">
    <source src="film_WebM.webm" type="video/webm">
    <source src="film_HEVC.mp4" type="video/mp4">
    <source src="film.mp4" type="video/mp4">
</video>
```
In this example Safari always plays h264 if it is a source but will otherwise happily play HEVC. MP4 seems to be the required container and hence mime-type.
